runit/kopano-server/run

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+set -euo pipefail
+
+source ../kopano-helper.sh
+
+ensure_config server.cfg
+ensure_config ldap.cfg
+
+exec 2>&1
+exec /usr/sbin/kopano-server -F
```

Comment:
Do we need to start it as root? Imo it is better to start is as user kopano directly.

Reply:
Unfortunately at the moment it needs to stay like this, but there is also little harm since kopano-server reexecs itself and then runs as the user kopano.

When directly running the binary as the user kopano the following is logged:

```
root@76c3f3418c84:/etc/kopano# /sbin/setuser kopano /usr/sbin/kopano-server -F
2021-06-09T09:39:22.804711: [info   ] Audit logging not enabled.
2021-06-09T09:39:22.804748: [=======] Starting kopano-server version 9.0.13 (pid 833 uid 999)
2021-06-09T09:39:22.804774: [info   ] Using epoll events
2021-06-09T09:39:23.189252: [warning] K-1558: Unable to limit socket 0.0.0.0%lo:236 to lo: Operation not permitted
2021-06-09T09:39:23.189270: [info   ] Listening on 0.0.0.0%lo:236 (fd 7)
2021-06-09T09:39:23.575050: [warning] K-1558: Unable to limit socket [::]%lo:236 to lo: Operation not permitted
2021-06-09T09:39:23.575064: [info   ] Listening on [::]%lo:236 (fd 8)
2021-06-09T09:39:23.961872: [debug  ] K-1562: changed owner "/var/run/kopano/prio.sock" to kopano:kopano mode 660
2021-06-09T09:39:23.961885: [info   ] Listening on unix:/var/run/kopano/prio.sock (fd 9)
2021-06-09T09:39:24.347589: [debug  ] K-1562: changed owner "/var/run/kopano/server.sock" to kopano:kopano mode 666
2021-06-09T09:39:24.347601: [info   ] Listening on unix:/var/run/kopano/server.sock (fd 10)
2021-06-09T09:39:24.349499: [debug  ] Preparing relaunch with allocator lib libtcmalloc_minimal.so.4
2021-06-09T09:39:24.352692: [debug  ] Reexecing /usr/sbin/kopano-server
2021-06-09T09:39:24.365252: [info   ] Audit logging not enabled.
2021-06-09T09:39:24.365289: [=======] Starting kopano-server version 9.0.13 (pid 833 uid 999)
2021-06-09T09:39:24.365308: [info   ] Using epoll events
2021-06-09T09:39:24.368483: [warning] K-1558: Unable to limit socket 0.0.0.0%lo:236 to lo: Operation not permitted
2021-06-09T09:39:24.368496: [error  ] K-1559: bind 0.0.0.0%lo:236: Address already in use
2021-06-09T09:39:24.371630: [warning] K-1558: Unable to limit socket [::]%lo:236 to lo: Operation not permitted
2021-06-09T09:39:24.371638: [error  ] K-1559: bind [::]%lo:236: Address already in use

An error occurred: no access (0xffffff9e). Please check logfile file:- for details.
```